python/ml4ir/base/config/dynamic_args.py

```diff
@@ -28,6 +28,11 @@ def cast_dynamic_val(val: str):
                 # int
                 return int(val)
         elif val.isalnum():
+            # bool
+            if val.lower() == "true":
+                return True
+            elif val.lower() == "false":
+                return False
```

Comment:
This allows me to not have 10 different feature configs for doing feature ablation studies. I can just set `--feature_config.features.feature_name.trainable false` and the feature will be removed from the model.

Reply:
(I understand this is not related to the PR, but leaving here as it is a tiny change)